pkgs/development/python-modules/stestr/tests.nix

```diff
@@ -0,0 +1,27 @@
+{ buildPythonPackage
+, stestr
+, coverage, ddt, doc8/*, openstack-hacking*/, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "stestr-tests";
+  inherit (stestr) version meta;
+
+  src = stestr;
+
+  dontConfigure = true;
+  dontBuild = true;
+  dontInstall = true;
+
+  checkInputs = [
+    coverage
+    ddt
+    doc8
+    # openstack-hacking # To be enabled after `openstack-hacking` is merged.
+    sphinx
+  ];
+
+  # checkPhase = ''
+  #   stestr
+  # '';
+}
```

Comment:
Admirable that you've done this, but until the other package has been merged, this is probably better left out. You'd need another PR or commit to come back and enable these tests anyway. You could just add them then.

Reply:
Enabling is less work than typing it all. It acts as a placeholder. I'm trying to port OpenStack and currently I'm upstreaming parts. As I get parts in place I will be enabling tests. The other option I had was to make a huge PR. But there was also a huge PR that failed. I prefer leaving these comments parts. It has just proved useful, with your help of course.

Reply:
Have you considered tracking your OpenStack work as a project? https://github.com/NixOS/nixpkgs/projects